sw/device/silicon_creator/rom_ext/imm_section/README.md

```diff
@@ -0,0 +1,78 @@
+# RomExt Immutable Section Release Guide
+
+## Bump the release version
+
+Edit the `IMM_SECTION_VERSION_{MAJOR,MINOR}"` line in the bazel file.
```

Comment:
After editing, a commit and PR should be made so that one builds from a clean tree.

Building / stamping from a dirty tree should only be done for testing purposes.